DigitalLearningSolutions.Data/Services/SectionContentService.cs

```diff
@@ -57,23 +66,43 @@ ELSE CAST(SUM(aspProgress.TutStat) * 100 AS FLOAT) / (COUNT(Tutorials.TutorialID
                         LEFT JOIN aspProgress
                             ON aspProgress.TutorialID = CustomisationTutorials.TutorialID
                             and aspProgress.ProgressID = Progress.ProgressID
+                        LEFT JOIN (SELECT COUNT(AssessAttemptID) AS AttemptsPL,
+                            AssessAttempts.ProgressID,
+                            AssessAttempts.SectionNumber,
+                            COALESCE (MAX(ISNULL(CAST(AssessAttempts.Status AS Integer), 0)), 0) AS PLPassed
```

Comment:
What exactly are you trying to do here? 🙂

Reply:
I'm not 100% sure on this one, I took the select statement from the stored procedure. Wasn't entirely sure how it worked

Reply:
I see, I've found it now. So we're trying to assentially OR together all the Status fields (which are booleans). However that's sadly a pain to do in SQL 🙁 
But I think we can fix this and the type problem in one go:
1. Change this to count the number of passes instead of checking if there is one, e.g. `SUM(CAST(Status AS INT))`
2. Accept an int in the constructor of the section content, but make the constructor change it to a boolean (true if the sum is greater than 0).

Does that make sense? 🙂